main.cpp

```diff
@@ -50,26 +51,387 @@ If you need inspiration for what to write, take a look at previously approved st
  you can resolve any [-Wdeprecated] warnings by adding -Wno-deprecated to list of compiler arguments in the .replit file. all of the "-Wno" in that file are compiler arguments.
  You can resolve any [-Wpadded] warnings by adding -Wno-padded to the list of compiler arguments in the .replit file. all of the "-Wno" in that file are compiler arguments.
  */
-
+#include <iostream>
 /*
  UDT 1:
  */
+ struct Computer
+{
+    int RAM;
```

Comment:
Fix this indentation
`RAM` is a name, not a property.  You're using it to indicate a quantity, so the name should reflect that:
`amountOfRAM` is what you should use here
- [ ] revise